plugins/Actions/API.php

```diff
@@ -445,28 +452,15 @@ protected function doFilterPageDatatableSearch($callBackParameters, $table, $sea
      * Common filters for all Actions API
      *
      * @param DataTable|DataTable\Simple|DataTable\Map $dataTable
-     * @param bool $expanded
      */
-    protected function filterActionsDataTable($dataTable, $expanded = false)
+    private function filterActionsDataTable($dataTable)
     {
         // Must be applied before Sort in this case, since the DataTable can contain both int and strings indexes
         // (in the transition period between pre 1.2 and post 1.2 datatable structure)
-        $dataTable->filter('ReplaceColumnNames');
```

Comment:
This all moved into a separate filter so it is better reusable and would be also better testable